main.js

```diff
@@ -989,21 +995,39 @@ function _select_Gateway() { // Check for Gateway used to
         settings["charcount"] = 99;
     }
 
+	var tasklist;
+	// Profession priority list by order
+	var defaultTasklist = [

```

Comment:
Well, something like Alchemy should be close to the top because it will only need one slot most of the time. Leadership should be last so that it can be set to soak up any slots that have not been taken up by all other professions. I always leave Leadership at 9 slots even if I am using up slots for another profession.

I just feel like professions that take up fewer slots should be set first and leadership should always be last to soak up any slots that are left over. Personal preference or not, I would like to know if any other order would be more functional.